### PR TITLE
Debranding work and some README.md changes

### PR DESCRIPTION
**Pull request by Samisafool**

### CODE REVIEW

File: cmake/UnitTest/test.rc [hunk 1]
```diff
@@ -14,7 +14,7 @@ BEGIN
         BEGIN
                 BLOCK "000004b0"
                 BEGIN
-                        VALUE "CompanyName", "MultiMC Contributors"
+                        VALUE "CompanyName", "UltimMC Contributors"
```
Comment (maximmasterr) [inline cmake/UnitTest/test.rc:17]: Nope, most of code written by MultiMC so here should be MultiMC contributors

File: launcher/package/ubuntu/multimc/usr/share/metainfo/multimc.metainfo.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <summary>Manage Minecraft instances with ease</summary>
     <description>
         <p>Overview</p>
-        <p>MultiMC is a free, open source launcher for Minecraft. It allows you to have multiple, cleanly separated instances of Minecraft (each with their own mods, texture packs, saves, etc) and helps you manage them and their associated options with a simple and powerful interface.</p>
+        <p>UltimMC is a free, open source launcher for Minecraft. It allows you to have multiple, cleanly separated instances of Minecraft (each with their own mods, texture packs, saves, etc) and helps you manage them and their associated options with a simple and powerful interface.</p>
```
Comment (maximmasterr) [inline launcher/package/ubuntu/multimc/usr/share/metainfo/multimc.metainfo.xml:9]: Add here note about light-fork of MultiMC with cracked accounts and accounts from ely.by

File: launcher/package/ubuntu/multimc/usr/share/man/man1/multimc.1 [hunk 3]
```diff
@@ -28,20 +28,20 @@
...
 .SH "NAME"
-multimc \- a launcher and instance manager for Minecraft\&.
+ultimmc \- a launcher and instance manager for Minecraft\&.
 .SH "SYNOPSIS"
 .sp
 \fBmultimc\fR [\fIOPTIONS\fR]
 .SH "DESCRIPTION"
 .sp
-MultiMC is a custom launcher for Minecraft that allows you to easily manage multiple installations of Minecraft at once\&. It also allows you to easily install and remove mods by simply dragging and dropping\&. Here are the current features of MultiMC\&.
+UltimMC is a custom launcher for Minecraft that allows you to easily manage multiple installations of Minecraft at once\&. It also allows you to easily install and remove mods by simply dragging and dropping\&. Here are the current features of UltimMC\&.
```
Comment (maximmasterr) [inline launcher/package/ubuntu/multimc/usr/share/man/man1/multimc.1:37]: Add here note about light-fork of MultiMC with cracked accounts and accounts from ely.by

File: doc/multimc.1.txt [hunk 4]
```diff
@@ -5,31 +5,31 @@ MULTIMC(1)
...
 SYNOPSIS
 --------
-*multimc* ['OPTIONS']
+*ultimmc* ['OPTIONS']
 
 
 DESCRIPTION
 -----------
-MultiMC is a custom launcher for Minecraft that allows you to easily manage
+UltimMC is a custom launcher for Minecraft that allows you to easily manage
```
Comment (maximmasterr) [inline doc/multimc.1.txt:18]: Add here note about light-fork of MultiMC with cracked accounts and accounts from ely.by

**Review by maximmasterr — CHANGES_REQUESTED** [review]

File: cmake/UnitTest/test.rc (hunk 1 above)
Comment (Samisafool) [reply cmake/UnitTest/test.rc:17]: Technically all the code in MultiMC is also part of UltimMC, so UltimMC contributors includes MultiMC contributors?

Comment (maximmasterr) [reply cmake/UnitTest/test.rc:17]: I guess no, people who contributed to MultiMC contributed to MultiMC bot UltimMC so they MultiMC contributors

Comment (Samisafool) [reply cmake/UnitTest/test.rc:17]: okay, will solve

**Comment by Samisafool** [comment]
Don't merge yet, will debrand more

**Comment by Samisafool** [comment]
I found this one folder launcher/resources/multimc/ which I'm not sure is ok to change. lmk if you find out how to.

**Comment by Samisafool** [comment]
Alright I'm done

**Comment by Sebastian-byte** [comment]
Again, for debrand work, do it directly in the upstream launcher, doing it only here will only cause conflicts and make maintaining it more painful.

**Comment by Samisafool** [comment]
It is not possible to debrand it upstream without creating similar conflicts, because you cannot use variables in the manpage or package description

**Comment by Sebastian-byte** [comment]
> It is not possible to debrand it upstream without creating similar conflicts, because you cannot use variables in the manpage or package description or github issue templates

I don't really get what you mean by "conflicts" there, I'm talking about Git conflicts, and about that, in that case you should use the generic name, "Launcher"

**Comment by Samisafool** [comment]
> I don't really get what you mean by "conflicts" there, I'm talking about Git conflicts, and about that, in that case you should use the generic name, "Launcher"

Did you see what I changed? There are URLs too, which I changed to this repo.

**Comment by Sebastian-byte** [comment]
> Did you see what I changed? There are URLs too, which I changed to this repo.

Then consider making name changes in the upstream repository and then URLs changes in this repository later.

**Pull request closed without merging**